source/includes/_lips_events.md

```diff
@@ -0,0 +1,380 @@
+### **Events**
+<a name="lips_events"></a>
+
+# Events API
+
+Also referred to as "contact interactions", *events* are activities associated with a contact.  The event may be a call, a form post, or a chat event recorded by capture.  It could also be created by posting to this API.
+
+### Resource Overview
+
+| Method | URI Format |
+|---|---|
+| GET `index` | /fpd_events? |
+| GET `index` | /fpd_events/#id |
+| GET `index` | /contacts/#id/events? |
```

Comment:
The ? at the end of the events route should be removed.  It's not part of the route